BrainPortal/app/models/cluster_task.rb

```diff
@@ -1904,8 +1916,9 @@ def docker_present?
   # Returns the command line(s) associated with the task, wrapped in
   # a Docker call if a Docker image has to be used.
   def docker_commands
+    workDir  = self.container_working_directory || '${PWD}'
```

Comment:
I'm not asking you to change it, but in general avoid using variables with `namesUsingInternalCapitals`, it's not the Ruby way.